src/core/qgsvectorlayerutils.h

```diff
@@ -68,6 +105,20 @@ class CORE_EXPORT QgsVectorLayerUtils
                                      const QgsAttributeMap &attributes = QgsAttributeMap(),
                                      QgsExpressionContext *context = nullptr );
 
+    /**
+     * Duplicates a feature and it's children (one level deep). It calls CreateFeature, so
+     * default values and constraints (e.g., unique constraints) will automatically be handled.
+     * The duplicated feature will be automatically inserted into the layer.
+     * \param layer
+     * \param feature the active feature
+     * \param project
+     * \param depth the higher this number the deeper the level - With depth > 0 the children of the feature are not duplicated
+     * \param duplicateFeatureContext stores all the layers and the featureids of the duplicated features (incl. children)
```

Comment:
Either use `\param` and document every parameter with an explanation or just clarify some parameters with `\a` inline.